aggsender/db/migrations/0001.sql

```diff
@@ -8,5 +8,8 @@ CREATE TABLE certificate_info (
     status                      INTEGER NOT NULL,
     new_local_exit_root         VARCHAR NOT NULL,
     from_block                  INTEGER NOT NULL,
-    to_block                    INTEGER NOT NULL
+    to_block                    INTEGER NOT NULL,
+    created_at                  INTEGER NOT NULL,
+	updated_at                  INTEGER NOT NULL,
```

Comment:
This one is misaligned (tab instead of 4 spaces):

```suggestion
    updated_at                  INTEGER NOT NULL,
```

Reply:
fix: https://github.com/0xPolygon/cdk/pull/175/commits/f5f86ffc7c0eeae58e6a7c2b1979af6f06ca40cf